app/policies/spam_policy.rb

```diff
@@ -0,0 +1,9 @@
+class SpamPolicy < ApplicationPolicy
```

Comment:
I know these are called the spam pages, but maybe this should be the `ModeratedWorkPolicy` instead, so it's more obvious it's not related to spam comments. Or... should we use the `UserCreationPolicy` for this, considering it already controls the ability to mark works as spam? 

I think I'm going to solicit some additional opinions on this.